packages/runtime-types/index.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module '*.st.css' {
```

Comment:
having a third party for this definition feels a bit odd. especially since it affects `*.st.css`. we don't usually let third parties augment `*.xyz` for us.

most typescript projects out there don't use "types", as it's not enabled by default for `tsc --init`.
if such projects turn it on, they will also have to specify other implicitly loaded types (node; jest; webpack-env; etc), making integration into existing projects harder.

Reply:
Alright, I agree that it would be best to have the types without changing the "types" config but keeping it like today has many downsides. The other solution is to create a definitely typed package called something like `@types/stylable-runtime` / `@types/stylable-stylesheet`

@AviVahl  @barak007 wdyt?